tgstation.dme

```diff
@@ -4957,4 +4957,6 @@
 #include "modular_splurt\code\modules\vending\snack.dm"
 #include "modular_splurt\code\modules\vending\wardrobes.dm"
 #include "tools\Redirector\textprocs.dm"
+#include "modular_splurt\code\game\objects\items\lewd_russian_revolver\lewd_russian_revolver.dm" // At least 10 years in purgatory for this shit, thank you tibbets.
+
```

Comment:
Please don't manually include files. Delete this line and instead click the Unticked button at the bottom of your editor when you open modular_splurt/code/game/objects/items/lewd_russian_revolver/lewd_russian_revolver.dm to let vscode include the file on its own. You may also turn on automatic file ticking so it includes new files automatically